src/views/Download.vue

```diff
@@ -151,14 +177,25 @@
       },
 
       selectedLayerData() {
-        return this.activeLayers.find(layer => layer.id === this.selectedLayer)
+        return this.downloadLayers.map(id => this.activeLayers.find(layer => layer.id === id))
       },
 
       selectionCoordinates() {
         return this.drawnFeatureCoordinates.toString().replace(/,/g, ' ')
       },
     },
 
+    watch: {
+      rasterLayerIds: {
+        handler: function (val, oldVal) {
+          if (val !== oldVal) {
```

Comment:
Do you need to perform this check here? The watcher should only get triggered when these two differ right?

Sidenote, why do you use the handler pattern instead of `rasterLayerIds() {}`?

Reply:
Yes, it should only be triggered if the values are different. And yes, the shorthand is fine too.